lib/astute/orchestrator.rb

```
@@ -183,31 +182,51 @@ def download_release(up_reporter, task_id, release_info)
       deploy_engine_instance.deploy(nodes, attrs)
       proxy_reporter.report({'status' => 'ready', 'progress' => 100})
     end
-    
+
+    def check_redhat_credentials(reporter, task_id, credentials)
+      ctx = Context.new(task_id, reporter)
+      begin
+        Astute::RedhatChecker.new(ctx, credentials).check_redhat_credentials
+      rescue Exception => e
```

Do we really need catch all exception including SyntaxError, LoadError and Interrupt? Maybe enough to catch only StandartError?